getting-started.md

```diff
@@ -0,0 +1,142 @@
+
+**For MacOS:**
+
+Download `vegawallet-darwin-amd64.zip`
+
+When you open the file, you may need to change your system preferences for this specific instance, in order to run Vega Wallet. If you open the file from downloads, you may get a message saying “vegawallet-darwin-amd64” cannot be opened because it is from an unidentified developer.
+
+Click on the (?) help button, which will open a window that links you to the System Preferences, and instructs you how to allow this software to run.
+
+You’ll need to go to System Preferences > Security & Privacy > General, and choose “Open Anyway”.
+
+**For Windows:**
+
+Download `vegawallet-windows-amd64.zip`
+
+You may need to change your system preferences for this specific instance, in order to run Vega Wallet. If you open the file from downloads, you may get a message from Windows Defender saying “vegawallet-windows-amd64” cannot be opened because it is from an unidentified developer.
+
+Click on the (More info) text, which will reveal a button to "Run anyway".
+
+**For Linux:**
+
+Download `vegawallet-linux-amd64.zip`
+
+## Generate key pair and credentials
+
+### Execute the program
+
+> Tip: You'll need to run the commands from the directory you've saved the wallet file in. Use the command `pwd` to find out where your terminal is looking in the file system. Use the command `cd` and the path/to/wallet/directory to tell the command line where to find the file. 
+
+> Tip: You can use the tab key to auto-fill the name of the file, after you type the first few characters.
+
+**MacOS & Linux**
+
+Open a new terminal. Type
+
+```console
+./go-wallet
+```
+to execute the program.
+
+**Windows**
+
+Open a new command prompt. Type
+
+```console
+go-wallet
+```
+to execute the program.
+
+> Tip: You can see a list of available commands by running  `./go-wallet -h` on MacOS and Linux, or `go-wallet -h` on Windows.
+
+### Create name and passphrase
+Next, **create a user name and passphrase** for your Wallet, and **create a public and private key** (genkey). 
+
+Replace "YOUR_CUSTOM_USERNAME" (below) with your chosen username:
+
+**MacOS & Linux**
+
+```console
+./go-wallet genkey -n "YOUR_CUSTOM_USERNAME"
+```
+
+**Windows**
+
+```console
+go-wallet genkey -n "YOUR_CUSTOM_USERNAME"
+```
+
+It will then prompt you to **input a passphrase**, and then **confirm that passphrase**. You'll use this username and passphrase to login to Vega Console. (Instructions on connecting to Console are below.)
+
+The genkey command in that instruction will generate public and private keys for the wallet, at the same time as creating a user name.
+
+You’ll see an output with a "mnemonic" and a public and private key. DO NOT SHARE YOUR MNEMONIC OR YOUR PRIVATE KEY.
+
+**The mnemonic acts as a backup key, from which the wallet can restore all your keys.** Keep it safe and secret. If lost, you won't be able to retrieve your keys. Anyone who has this mnemonic will be able to use your keys.
```

Comment:
> Anyone who has this mnemonic will be able to use your keys.

I find this phrasing about the mnemonic much better than the one I've written in the `cmd/README.md`  (_"If stolen, the thief..."_) . Would you mind to update it there as well ?